ui/app/services/control-group.js

```diff
@@ -135,6 +135,10 @@ export default Service.extend({
     return {
       type: 'error-with-html',
       content: lines.join('\n'),
+      href,
+      token,
+      accessor,
+      creation_path,
```

Comment:
<img width="712" alt="Screenshot 2024-08-30 at 11 14 52 AM" src="https://github.com/user-attachments/assets/77381638-bae4-48d6-83cb-3058afb04440">
Adding these here meant that I could remove the {{{ triple curly brackets }}}  which linter doesn't like because they're unsafe, see commit here: 
https://github.com/hashicorp/vault/pull/28232/commits/3f5f383f8137a61e89f54b5df9a8cbdf27c3de66#diff-e664502e7b8b3559d82c13c61fdf85514717e3e66c9d3530ec09c29abeec291bL83-L84